test/suites/drift/suite_test.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("Drift", Label("AWS"), func() {
 		})
```

Comment:
I think we should replace the `selectCustomAMI` function in this file with the `GetCustomAMI` function in the `aws/expectations.go` file so that we aren't duplicating the code throughout.

Reply:
Will implement in a follow up PR